### PR TITLE
Added Database factories and seeder for all models

### PR DESCRIPTION
**Pull request by ddhira123**
Changes summary:

- [x]  Add factories for all models.
- [x]  Add `timestamp` as Database Column for the table: `Image`.
- [x]  Add Database seeders for all models.
- [x] Upgrade dependencies.
- [ ] Create relationships between entities in Seeders.

### CODE REVIEW

File: database/migrations/2020_11_11_112707_images.php [hunk 1]
```diff
@@ -18,8 +18,8 @@ public function up()
         Schema::create($this->table_name, function (Blueprint $table) {
             $table->id();
             $table->string('alt');
-            $table->string('link');
             $table->string('url');
+            $table->timestamps();
```
Comment (mhilmyh) [inline database/migrations/2020_11_11_112707_images.php:22]: ini kenapa ada timestamp nya dan ?

File: database/seeders/EmailSeeder.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        Email::factory()->times(20)->create();
+    }
+}
```
Comment (mhilmyh) [inline database/seeders/EmailSeeder.php:19]: kayaknya email seeder di hapus aja deh dan

File: database/seeders/DatabaseSeeder.php [hunk 3]
```diff
@@ -14,5 +21,18 @@ class DatabaseSeeder extends Seeder
     public function run()
     {
         // $this->call('UsersTableSeeder');
+
+        $this->call([
+            UserSeeder::class,
+            ActivitySeeder::class,
+            EmailSeeder::class,
```
Comment (mhilmyh) [inline database/seeders/DatabaseSeeder.php:28]: Ini ga dipake lagi kan ya ?

File: database/factories/UserFactory.php [hunk 4]
```diff
@@ -22,8 +23,8 @@ class UserFactory extends Factory
     public function definition()
     {
         return [
-            'name' => $this->faker->name,
             'email' => $this->faker->unique()->safeEmail,
+            'password' => Str::random(15)
```
Comment (mhilmyh) [inline database/factories/UserFactory.php:27]: Ini passwordnya udah auto hash kan dhan ?

File: database/factories/EmailFactory.php [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+     */
+    public function definition()
+    {
+        return [
+            'message' => $this->faker->paragraph,
+            'sender_id' => $this->random20(),
+            'receiver_id' => $this->random20()
+        ];
+    }
+}
```
Comment (mhilmyh) [inline database/factories/EmailFactory.php:38]: ini kayaknya gadipake lagi

**Comment by mhilmyh** [comment]
nvm, ntar aja deh benerinnya biar ga lama

**Review by mhilmyh — APPROVED** [review]

File: database/seeders/DatabaseSeeder.php (hunk 3 above)
Comment (ddhira123) [reply database/seeders/DatabaseSeeder.php:28]: Oke, gw hapus ya

Comment (ddhira123) [reply database/seeders/DatabaseSeeder.php:28]: Oh udah di merge. Oke deh.

**Pull request merged**